src/services/adapters/activity-adapter/activity.adapter.ts

```diff
@@ -537,14 +537,29 @@ export class ActivityAdapter {
       `
     );
 
-    if (!result) {
-      throw new EntityNotFoundException(
-        `Unable to identify Collaboration for Whiteboard with ID: ${whiteboardID}`,
-        LogContext.ACTIVITY
+    if (!contributionResult) {
+      const [profileResult]: {
```

Comment:
As far as I see, this is `collaborationResult`, not `profileResult` - please rename.

Reply:
ok, got your point.